readthedocs/api/v2/views/integrations.py

```diff
@@ -83,7 +83,7 @@ def post(self, request, project_slug):
                 resp = {'detail': 'This project is currently disabled'}
                 return Response(resp, status=status.HTTP_406_NOT_ACCEPTABLE)
         except Project.DoesNotExist:
-            raise NotFound('Project not found')
+            raise NotFound('Project not found') from None
```

Comment:
probably I'll ignore these type of warnings, as it's just an esthetical change in the traceback.